pkg/agent/config.go

```diff
@@ -36,6 +38,16 @@ type Config struct {
 	// CacheActiveTimeout specifies the maximum duration that flows are kept in the accounting
 	// cache before being flushed for its later export
 	CacheActiveTimeout time.Duration `env:"CACHE_ACTIVE_TIMEOUT" envDefault:"5s"`
+	// Deduper specifies the deduper type. Accepted values are "none" (disabled) and "firstCome".
+	// When enabled, it will detect duplicate flows (flows that have been detected e.g. through
+	// both the physical and a virtual interface).
+	// "firstCome" will forward only flows from the first interface the flows are received from.
+	Deduper string `env:"DEDUPER" envDefault:"none"`
+	// DeduperFCExpiry specifies the expiry duration of the flows "firstCome" deduplicator. After
+	// a flow hasn't been received for that expiry time, the deduplicator forgets it. That means
+	// that a flow from a connection that has been inactive during that period could be forwarded
+	// again from a different interface.
+	DeduperFCExpiry time.Duration `env:"DEDUPER_FC_EXPIRY" envDefault:"30s"`
```

Comment:
Isn't the 30s default a bit too high causing unnecessary memory consumption?

For a same flow being captured going through 2 different interface on the same node, I would have thought that this was below the second. So may be a default value like 5s would be enough?

Reply:
Hmm since I want to be consistent between flows' flush periods, this should be > 1 cache flush periods. So probably we can set it, by default, to "CacheActiveTimeout * 2" (10s)